src/main/java/service/impl/ReportGeneratorImpl.java

```diff
@@ -0,0 +1,21 @@
+package service.impl;
+
+import db.Storage;
+import java.util.Map;
+import service.ReportGenerator;
+
+public class ReportGeneratorImpl implements ReportGenerator {
```

Comment:
The class implementation should not begin with an empty line. Please remove the redundant empty line at the start of the class definition.